client/src/shared/validation/index.ts

```diff
@@ -60,13 +60,13 @@ export const MatterFormSchema = Yup.object().shape({
 })
 
 export const CourtAppearanceSchema = Yup.object().shape({
-  date: Yup.string().max(255).required('Must be a valid date'),
-  next_court_date: Yup.string().max(255).required('Must be a valid date'),
-  time: Yup.string().max(255).required('Must be a valid time'),
-  court: Yup.string().max(255).required('This field is required'),
-  judicial_officer: Yup.string().max(255).required('This field is required'),
-  orders: Yup.string().required('This field is required'),
-  other_notes: Yup.string()
+    date: Yup.string().required('This field is required'),
+    next_court_date: Yup.string().required('This field is required'),
```

Comment:
Next Court Date is required

Reply:
@jsvelte It's already indicated the the `next_court_date` field is required